### PR TITLE
add userguide about imagePullSecrets #29122384

### PR DESCRIPTION
**Pull request by jiaqianjing**

### CODE REVIEW

File: docs/userguide/26-submitjob-with-private-registry.md [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+               "python /root/code/mnist-pytorch/mnist.py --epochs 10 --backend nccl --dir /mnist_data/pytorch_data/logs --data /mnist_data/pytorch_data/"
+           configmap/pytorch-git-with-secret-pytorchjob created
+           configmap/pytorch-git-with-secret-pytorchjob labeled
+           service/pytorch-git-with-secret-tensorboard created
+           deployment.apps/pytorch-git-with-secret-tensorboard created
+           pytorchjob.kubeflow.org/pytorch-git-with-secret created
+           INFO[0001] The Job pytorch-git-with-secret has been submitted successfully
+           INFO[0001] You can run `arena get pytorch-git-with-secret --type pytorchjob` to check the job status
+           ```
+       2. get job details.
```
Comment (cheyang) [inline docs/userguide/26-submitjob-with-private-registry.md:167]: Get the details of the job

Comment (jiaqianjing) [reply docs/userguide/26-submitjob-with-private-registry.md]: updated

**Comment by jiaqianjing** [comment]
> Thanks for your contribution @jiaqianjing ! Looks fine in general except several grammar and spelling issues. And I think it's better you combine #342 , #344 , #346 and #347 into one PR.

3q.

File: docs/userguide/26-submitjob-with-private-registry.md [hunk 2]
```diff
@@ -0,0 +1,180 @@
+# Using a Private Registry for jobs
```
Comment (cheyang) [inline docs/userguide/26-submitjob-with-private-registry.md:1]: Submit the training jobs with ImagePullSecrets

File: docs/userguide/26-submitjob-with-private-registry.md [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+
+You can use a private registry when submiting jobs(include tensorboard images).
+Suppose the images in the following list from your private registry.
```
Comment (jasonliu747) [inline docs/userguide/26-submitjob-with-private-registry.md:4]: Assume the following images are in your private rigistry.

File: docs/userguide/26-submitjob-with-private-registry.md [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+              --logdir=/mnist_data/tf_data/logs \
+              --image-pull-secrets=lumo-secret \
+              "python code/tensorflow-sample-code/tfjob/docker/mnist/main.py --log_dir /mnist_data/tf_data/logs  --data_dir /mnist_data/tf_data/"
+    configmap/tf-git-with-secret-tfjob created
+    configmap/tf-git-with-secret-tfjob labeled
+    service/tf-git-with-secret-tensorboard created
+    deployment.apps/tf-git-with-secret-tensorboard created
+    tfjob.kubeflow.org/tf-git-with-secret created
+    INFO[0001] The Job tf-git-with-secret has been submitted successfully
+    INFO[0001] You can run `arena get tf-git-with-secret --type tfjob` to check the job status
```
Comment (jasonliu747) [inline docs/userguide/26-submitjob-with-private-registry.md:73]: Redundant.

File: docs/userguide/26-submitjob-with-private-registry.md [hunk 5]
```diff
@@ -0,0 +1,180 @@
...
+          --tensorboard-image=registry.cn-huhehaote.aliyuncs.com/lumo/tensorflow:1.12.0-devel \
+          --image-pull-secrets=lumo-secret  \
+          "mpirun python code/benchmarks/scripts/tf_cnn_benchmarks/tf_cnn_benchmarks.py --model resnet101 --batch_size 64 --variable_update horovod --train_dir=/training_logs --summary_verbosity=3 --save_summaries_steps=10"
+   configmap/mpi-dist-with-secret-mpijob created
+   configmap/mpi-dist-with-secret-mpijob labeled
+   service/mpi-dist-with-secret-tensorboard created
+   deployment.apps/mpi-dist-with-secret-tensorboard created
+   mpijob.kubeflow.org/mpi-dist-with-secret created
+   INFO[0001] The Job mpi-dist-with-secret has been submitted successfully
+   INFO[0001] You can run `arena get mpi-dist-with-secret --type mpijob` to check the job status
```
Comment (jasonliu747) [inline docs/userguide/26-submitjob-with-private-registry.md:123]: Redundant.

File: docs/userguide/26-submitjob-with-private-registry.md [hunk 6]
```diff
@@ -0,0 +1,180 @@
...
+       --logdir=/mnist_data/pytorch_data/logs \
+       --image-pull-secrets=lumo-secret \
+       "python /root/code/mnist-pytorch/mnist.py --epochs 10 --backend nccl --dir /mnist_data/pytorch_data/logs --data /mnist_data/pytorch_data/"
+   configmap/pytorch-git-with-secret-pytorchjob created
+   configmap/pytorch-git-with-secret-pytorchjob labeled
+   service/pytorch-git-with-secret-tensorboard created
+   deployment.apps/pytorch-git-with-secret-tensorboard created
+   pytorchjob.kubeflow.org/pytorch-git-with-secret created
+   INFO[0001] The Job pytorch-git-with-secret has been submitted successfully
+   INFO[0001] You can run `arena get pytorch-git-with-secret --type pytorchjob` to check the job status
```
Comment (jasonliu747) [inline docs/userguide/26-submitjob-with-private-registry.md:165]: Redundant.

**Pull request merged**